core/src/main/java/org/infinispan/functional/EntryView.java

```diff
@@ -107,6 +108,88 @@ private EntryView() {
     * @since 8.0
     */
    @Experimental
-   public interface ReadWriteEntryView<K, V> extends ReadEntryView<K, V>, WriteEntryView<V> {}
+   public interface ReadWriteEntryView<K, V> extends ReadEntryView<K, V>, WriteEntryView<V> {
+      /**
+       * Sets this value to the result of <code>updateFunction</code> applied
+       * to the current value, or <code>null</code> if absent.
+       * If <code>updateFunction</code> returns <code>null</code>, the mapping
+       * is removed.
+       *
+       * @param updateFunction
+       * @return <code>null</code>
+       * @since 9.1
+       */
+      default Void update(Function<? super V, ? extends V> updateFunction) {
```

Comment:
I don't understand how $SUBJECT applies to the changes. Should you be using SerializableFunction here ?